play/src/main/scala/com/managementapp/authentication/models/UserTable.scala

```diff
@@ -0,0 +1,15 @@
+package com.managementapp.authentication.models
+
+import slick.lifted.Tag
+import slick.driver.MySQLDriver.api._
+
+
+case class User(email: String, password: String)
```

Comment:
Oh right, my bad. I forgot that db table definitions are not necessarily the same as the data being transfered over http